### PR TITLE
Fix CAUTION note

### PR DESCRIPTION
**Pull request by ghogen**

### CODE REVIEW

File: msbuild-api-docs/xml/Microsoft.Build.Framework/CustomBuildEventArgs.xml [hunk 1]
```diff
@@ -36,9 +36,9 @@
       <format type="text/markdown"><![CDATA[
             ## Remarks
             > [!CAUTION]
-            In .NET 8 and later and Visual Studio 17.8 and later, this type is deprecated; instead use <see cref="ExtendedCustomBuildEventArgs"/>.
-            For more information, see <see href="https://learn.microsoft.com/dotnet/core/compatibility/sdk/8.0/custombuildeventargs"/>
-            For recommended replacement, see <see href="https://learn.microsoft.com/dotnet/core/compatibility/sdk/8.0/custombuildeventargs#recommended-action" />.
+            > In .NET 8 and later and Visual Studio 17.8 and later, this type is deprecated; instead use <see cref="ExtendedCustomBuildEventArgs"/>.
```
Comment (ghogen) [inline msbuild-api-docs/xml/Microsoft.Build.Framework/CustomBuildEventArgs.xml]:
```suggestion
            > In .NET 8 and later and Visual Studio 17.8 and later, this type is deprecated; instead use [ExtendedCustomBuildEventArgs](/dotnet/api/microsoft.build.framework.extendedcustombuildeventargs).
```

File: msbuild-api-docs/xml/Microsoft.Build.Framework/CustomBuildEventArgs.xml [hunk 2]
```diff
@@ -36,9 +36,9 @@
...
+            > For more information, see <see href="https://learn.microsoft.com/dotnet/core/compatibility/sdk/8.0/custombuildeventargs"/>
```
Comment (ghogen) [inline msbuild-api-docs/xml/Microsoft.Build.Framework/CustomBuildEventArgs.xml]:
```suggestion
            > For more information, see [MSBuild custom derived build events deprecated](/dotnet/core/compatibility/sdk/8.0/custombuildeventargs)
```

File: msbuild-api-docs/xml/Microsoft.Build.Framework/CustomBuildEventArgs.xml [hunk 3]
```diff
@@ -36,9 +36,9 @@
...
+            > For recommended replacement, see <see href="https://learn.microsoft.com/dotnet/core/compatibility/sdk/8.0/custombuildeventargs#recommended-action" />.
```
Comment (ghogen) [inline msbuild-api-docs/xml/Microsoft.Build.Framework/CustomBuildEventArgs.xml]:
```suggestion
            > For recommended replacement, see [Recommended action](/dotnet/core/compatibility/sdk/8.0/custombuildeventargs#recommended-action).
```

File: msbuild-api-docs/xml/Microsoft.Build.Framework/CustomBuildEventArgs.xml [hunk 4]
```diff
@@ -36,9 +36,8 @@
       <format type="text/markdown"><![CDATA[
             ## Remarks
             > [!CAUTION]
```
Comment (ghogen) [inline msbuild-api-docs/xml/Microsoft.Build.Framework/CustomBuildEventArgs.xml]:
```suggestion
            > [!WARNING]
```

**Review by rainersigwald — APPROVED** [review]

**Review by JanKrivanek — APPROVED** [review]

**Comment by ghogen** [comment]
@v-alje Can you merge and republish the MSBuild API docs? Thanks!

**Review by v-alje — APPROVED** [review]

**Comment by v-alje** [comment]
@ghogen Done.

**Pull request merged**